Source/Core/VideoCommon/DriverDetails.h

```diff
@@ -170,6 +170,15 @@ namespace DriverDetails
 		// We can't actually detect what the driver version is on Android, so until the driver version lands that displays the version in
 		// the GL_VERSION string, we will have to force vsync to be enabled at all times.
 		BUG_BROKENVSYNC,
+
+		// Bug: Mesa's I965 driver crashes when using ARB_shader_image_load_store
+		// Affected Devices: intel
+		// Started Version: 11.0.0
```

Comment:
That's mentioned a few lines down. 

I made it -1 anyway, as there will be git revisions before 11.0.0 where this is broken